vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -4,182 +4,198 @@ sidebarDepth: 2
-   - AMI type: `Amazon Linux 2`
-   - Instance type: `t3.medium`
-   - Review the Node Group scaling configuration 
-   - Set `Maximum size` to 5. This will be over-resourced for a `Getting Started` experience but will leave capacity for adding microservices to your cluster without modifying the Nodegroup.
-   - Click `Next`
-   - Review the Node Group network configuration.  
-   - `Subnets` - VPC subnets should already be setup and selected.
-   - Select `Configure SSH access to nodes`.  Follow the instructions to create a new SSH key pair if you don't already have one.
-   - Select `All` to allow all source IPs to access the nodes, or to set your own restrictions via Selected Security Groups.  
-   - Click `Next`
-   - Review your settings and then click `Create`. It may take a few minutes for the node to be created.
-7. Connect `kubectl` to the cluster
-   - *Note:* If this is a brand new setup you will need to login using the user you used to create your cluster in the console in the steps above. Make sure the users match.
-      - ```aws-configure``` (and then provide the Access key, etc.)
-   - Setup your kube context via ```aws eks --region region-code update-kubeconfig --name cluster_name```, e.g. ```aws eks --region us-east-2 update-kubeconfig --name cluster-1```
-   - More details and troubleshooting <https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html>
-   - Your current context should now be configured for your AWS cluster. Run the command below to check:
+
+Go to [Identity Management and Access on EKS](https://docs.aws.amazon.com/eks/latest/userguide/security-iam.html) for more details on roles.
+
+### Create the EKS Cluster
+1.   Add a cluster name and click `Create EKS cluster` (e.g. `YOUR-CLUSTER-1`).
+2. Select 1.21 for the Kubernetes version.
+3.  For `Cluster Service Role`, select the role you created above, `YOUR-EKS-ROLE`. If you choose a different role, it must have ELB permissions so the cluster can create a load balancer in `Networking`. 
+4. Click `Next`.
+5. Use the defaults for `Networking`(Step 2) and click `Next` 
+6. Use the defaults for `Configure Logging` (Step 3) and click `Next`.
+7. Review your settings and then click `Create`. Cluster provisioning usually takes between 10 and 15 minutes.
+
+See [Creating an Amazon EKS Cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) for more detailed information.
+
+### Add a Node Group to the Cluster
+1. Go to `Services` → `Elastic Kubernetes Service` → `Clusters` and select`YOUR-CLUSTER-NAME` 
+2. Go to `Compute` tab 
+3. Click `Add Node Group`
+4. Enter the following:
+      * `Name`: give your group a name, e.g. `YOUR-NODE-1`
+      * `Node IAM Role`: Select the cluster role you created above. If the role doesn't appear, verify that you added the extra policies to the role.
+      * Click `Next`
+5. For `Subnets` → the VPC subnets should already be setup and selected. Click `Next`
```

Comment:
the arrow here is misused. "`Subnets`: the VPC..." ??

period after "Click `Next`"